### PR TITLE
Admin view resumes page

### LINKED ISSUE #223: Admin - View User Resumes

### PR DESCRIPTION
**Pull request by SheepTester**
# Info

Closes #223.

view resumes page

## Changes

- view resumes page

# Type of Change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Logistics Change (A change to a README, description, or dev workflow setup like
      linting/formatting)
- [ ] Continuous Integration Change (Related to deployment steps or continuous integration
      workflows)
- [ ] Other: (Fill In) 

# Testing

I have tested that my changes fully resolve the linked issue ...

- [x] locally on Desktop.
- [ ] on the live deployment preview on Desktop.
- [ ] on the live deployment preview on Mobile.
- [ ] I have added new Cypress tests that are passing.

# Checklist

- [x] I have performed a self-review of my own code.
- [x] I have followed the style guidelines of this project.
- [x] I have documented any new functions in `/src/lib/*` and commented hard to understand areas
      anywhere else.
- [x] My changes produce no new warnings.

# Screenshots

image
image

### CODE REVIEW

File: src/components/admin/Resume/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            width={48}
+            height={48}
+            alt={`Profile picture for ${resume.user.firstName} ${resume.user.lastName}`}
+            className={styles.image}
+          />
+          <Typography variant="label/large" className={styles.name}>
+            {resume.user.firstName} {resume.user.lastName}
+          </Typography>
+          <p className={styles.info}>
+            {resume.user.graduationYear} {resume.user.major}
```
Comment (alexzhang1618) [inline src/components/admin/Resume/index.tsx]: maybe format it like
`{resume.user.major} ({resume.user.graduationYear})`?

**Review by alexzhang1618 — APPROVED** [review]
lgtm, should there be pagination on this page? on prod we have 44 resumes

down the line we should also support downloading all the resumes as a .zip, since that's the main use case of this page (to get all the resumes and send to companies as part of sponsorship package)

**Comment by SheepTester** [comment]
image

okay i added a zip all button

**Comment by SheepTester** [comment]
image
image

got approval

**Pull request merged**